examples/tutorials/advanced/working_with_panel.py

```diff
@@ -0,0 +1,141 @@
+r"""
+Dynamic data visualization using ``Panel``
+==========================================
+
+.. note::
+
+    Please run the following code examples in a notebook environment
+    otherwise the interactive parts of this tutorial will not work. You can
+    use the button "Download Jupyter notebook: panel_extension.ipynb" at
+    the bottom of this page to download this script as a Jupyter notebook.
+
+The library `Panel <https://panel.holoviz.org/index.html>`__ can be used to
+create interactive dashboards by connecting user-defined widgets to plots.
+``Panel`` can be used as an extension to Jupyter notebook / lab.
+
+This tutorial is split into three parts:
+
+- Make a static map
+- Make a dynamic map
```

Comment:
Maybe "dynamic" is not an appropriate word here:
```suggestion
- Make an interactive map
```